tests/gl/CMakeLists.txt

```diff
@@ -0,0 +1 @@
+add_subdirectory(ellipse)
```

Comment:
Make sure to add a comment to all newly added `CMakeLists.txt` files.

General idea is to have the relative path to the current file as comment at the begining.

As example, take a look at the following [CMakeLists.txt](https://github.com/MikroElektronika/mikrosdk_v2/blob/master/api/gl/lib/CMakeLists.txt#L1) file and add appropriate line with relative path to all CMake files you've added.

Reply:
Let's just be consistent with current CMakeLists.txt files.

So just add two hashtags for comment. `##`